README.md

```diff
@@ -1 +1,241 @@
 # avr-libstdcpp: libstdc++ port for avr-gcc
+
+`avr-libstdcpp` is a partial, non-fully-tested
+implementation of the C++ standard library and its STL.
+It is intended to be used with `avr-gcc`.
+
+Many features of modern C++11,14,17 and 20 are supported.
+
+`avr-gcc` does not have a complete C++ standard library nor does it include an STL implementation.
+The `avr-libstdcpp` port (even though not-fully-tested
+and only partially full/complete) will, nonetheless,
+be useful for those interested in making more comprehensive utilization
+of C++ --- including its standard library --- with a modern `avr-gcc` compiler.
```

Comment:
```suggestion
of C++, including its standard library, with a modern `avr-gcc` compiler.
```
Long dash in Markdown isn't rendered right and looks weird

Reply:
OK

Reply:
Done. Will be in next commit.